QgisModelBaker/gui/generate_project.py

```diff
@@ -484,16 +490,24 @@ def update_crs_info(self):
         self.crsSelector.setCrs(self.crs)
 
     def crs_changed(self):
-        if self.crsSelector.crs().authid()[:5] != 'EPSG:':
+        self.srs_auth = 'EPSG'  # Default
+        self.srs_code = 21781  # Default
+        srs_auth, srs_code = self.crsSelector.crs().authid().split(":")
+        if  srs_auth == 'USER':
             self.crs_label.setStyleSheet('color: orange')
             self.crs_label.setToolTip(
-                self.tr('Please select an EPSG Coordinate Reference System'))
-            self.epsg = 21781
+                self.tr('Please select a valid Coordinate Reference System.\nCRSs from USER are valid for a single computer and therefore, not recommended.\nA default EPSG:21781 will be used.'))
```

Comment:
Are they "not recommended" or "will 21781 be used"?
From the description it looks like using USER crs'es should be supported but here it looks like they are not accepted. How does that work exactly?

Reply:
I'm 50%-50% here. The only issue I see is that when using a `USER:XXXX`, the SRS code is not guaranteed to remain in different QGIS installations. Other than that, I'm fine if we remove the `USER` conditional.

Reply:
How about a warning (but letting the final choice to the user)? What does ili2db do with that anyway?

Reply:
ili2db's schema import fails if the SRS is not found in the spatial reference table of the target DB.
I'm fine with the warning. I'll adjust it.

Reply:
Let's remove the "not recommended" part. If ili2db does not support them we should not allow them.

Reply:
Done!